analyzers/tests/SonarAnalyzer.TestFramework.Test/Common/SourceGeneratorProviderTest.cs

```diff
@@ -36,8 +34,11 @@ public class SourceGeneratorProviderTest
 
     [TestMethod]
     public void RazorSourceGenerator_HasCorrectPath() =>
-        RazorSourceGenerator.FullPath
-            .Should().Be(Path.Combine(SourceGeneratorProvider.LatestSdkFolder(), "Sdks", "Microsoft.NET.Sdk.Razor", "source-generators", "Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators.dll"));
+        RazorSourceGenerator.FullPath.Should().EndWith(Path.Combine("Sdks", "Microsoft.NET.Sdk.Razor", "source-generators", "Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators.dll"));
```

Comment:
This test and the one above are now almost the same. That's not useful.

Reply:
I will keep the first one

Reply:
I would combine them to something like:
```cs
    public void SourceGenerators_ContainsRazorSourceGenerator() =>
        SourceGeneratorProvider.SourceGenerators.Should()
            .Contain(x => x.FullPath.EndsWith(Path.Combine("Sdks", "Microsoft.NET.Sdk.Razor", "source-generators", "Microsoft.CodeAnalysis.Razor.Compiler.SourceGenerators.dll")));
```